subsys/net/lib/fota_download/Kconfig

```diff
@@ -8,6 +8,8 @@ menuconfig FOTA_DOWNLOAD
 	bool "FOTA Download"
 	depends on DOWNLOAD_CLIENT
 	depends on DFU_TARGET
+	depends on SYS_HASH_FUNC32
+	depends on SYS_HASH_FUNC32_CHOICE_MURMUR3
```

Comment:
```suggestion
```

I don't think we need to care what is the default hash function. MURMUR3 is btw the default on
```

choice SYS_HASH_FUNC32_CHOICE
	prompt "Default system-wide 32-bit hash function"
	default SYS_HASH_FUNC32_CHOICE_MURMUR3
	help
	  The default system-wide 32-bit hash function is sys_hash32().
```

But after all, this code works with any of the selected hash functions.

Reply:
Fixed